test/SlowTests/Server/Replication/ReplicationCleanTombstones.cs

```diff
@@ -63,7 +63,7 @@ public async Task CleanTombstones(Options options)
 
                 Assert.Equal(1, WaitUntilHasTombstones(store2).Count);
 
-                EnsureReplicating(store1, store2);
```

Comment:
please sync the code between methods, this is a minefield now for others

Reply:
done :)